StyleCop.Analyzers/StyleCop.Analyzers.Test/OrderingRules/SA1204UnitTests.cs

```diff
@@ -294,6 +363,14 @@ static class TestClass2 { }
             var expected = this.CSharpDiagnostic().WithLocation(3, 14).WithArguments("internal", "classes");
 
             await this.VerifyCSharpDiagnosticAsync(testCode, expected, CancellationToken.None).ConfigureAwait(false);
+
+            var fixedCode = @"static class TestClass2 { }
+
+class TestClass1 { }
```

Comment:
:question: Why does this introduce an empty line, when none of the other tests above do. Is that because of the leading empty line in front?